build.gradle.kts

```diff
@@ -4,15 +4,22 @@ import com.vanniktech.maven.publish.SonatypeHost
 plugins {
   alias(libs.plugins.kotlin.serialization) apply false
   alias(libs.plugins.mavenPublish) apply false
+  id("org.jetbrains.compose") version "1.2.2" apply false
+  alias(libs.plugins.kotlin.multiplatform) apply false
+  alias(libs.plugins.kotlin.android) apply false
+  alias(libs.plugins.android.application) apply false
+  alias(libs.plugins.kotlin.native.cocoapods) apply false
```

Comment:
Need to follow-up on this... I got a TON of build issues related "unknown plugin exceptions" and " plugin is already on the classpath with an unknown version" when I tried adding the new `paging-runtime-compose` module. Only way I was able to fix all the errors was all these updates across the root project gradle and settings gradle